settings/authentication-personalization/sending-data.mdx

```diff
@@ -3,7 +3,7 @@ title: 'Sending Data'
 description: 'The types and shape of user data you can send to Mintlify'
```

Comment:
thoughts?
```suggestion
description: 'The types and shape of user data you can send to personalize Mintlify'
```

Reply:
I'm thinking "The shape of user data you can use to personalize Mintlify"

Reply:
sgtm!